mbedtls-sys/vendor/include/mbedtls/bn_mul.h

```diff
@@ -195,6 +196,25 @@
         : "b" (b)                           \
         : "rax", "rdx", "r8"                \
     );
+#else
+// Code copied from git https://github.com/ARMmbed/mbedtls.git commit 5f80040729cfbe0cf61a17cbceb64375d4b0acd3 crypto/include/mbedtls/bn_mul.h
```

Comment:
Why copy? The code is already here at the end of the file.

Reply:
Ok resolved, only changing preprocessor conditionals